tests/crypto/psa_crypto/src/aead.c

```diff
@@ -7,6 +7,10 @@
 #include <zephyr/ztest.h>
 #include <psa/crypto.h>
 
+#if defined CONFIG_TEST_WRAPPED_KEYS
```

Comment:
This is a generic test case for the PSA Crypto API. Implementation details for a specific device must not leak into it.